### ISSUE TITLE
Feature request: Comment in code

### ISSUE THREAD

**Issue by theotherp** [nhat-phan/merge-request-integration#3]
I'm not sure if this is even possible with IntelliJ and if yes you might have already considered this, but it would be great if we could leave code comments directly in the editor view.

Also, child comments to regular comments are displayed on the same level as their parent instead of a level below.

But it looks really great for a first version and so far it works mostly as I'd expected.

**Comment by nhat-phan** [comment]
@theotherp thank you for using the plugin.

I considered display comments in editor but I decided not do it yet because I want to keep the editor area as clean as possible, therefore I'll do it under an option in configuration.

Regarding child comment is displayed on the same level problem, I also think that multiple level comments is also not good because if you have too deep thread it's really hard to follow, isn't it? Then again I think an option in configuration is the best choice here.

Thanks again for your feedback, I prioritized the feature already, will do in [v2020.1.0](https://github.com/nhat-phan/merge-request-integration/projects/2)

**Comment by BlacKCaT27** [comment]
I second the notion, personally, at least as an option for sure. Perhaps you could just show an icon in the sidebar to indicate that a comment is there, which they can click on to expand?

**Comment by nhat-phan** [comment]
This feature takes me more than 3 weeks, luckily it's okay now, hope you guys will love it. Will available since v2020.1.2

![Screen Shot 2020-03-24 at 20 07 29](https://user-images.githubusercontent.com/34987391/77467079-9c0cc680-6e0b-11ea-9736-11101cdaa50d.png)

**Comment by theotherp** [comment]
Awesome!
